install/src/Help.hs

```diff
@@ -12,13 +12,23 @@ import           Version
 import           BuildSystem
 import           Cabal
 
+stackCommand :: TargetDescription -> String
+stackCommand target = "stack install.hs " ++ fst target
+
+cabalCommand :: TargetDescription -> String
+cabalCommand target = "cabal new-run install.hs --project-file install/shake.project " ++ fst target
```

Comment:
should we promote `v2-` since this is forward and backwards compatible?

Reply:
Yeah, cause we are supporting a cabal version that already has the `v2` prefixes

Reply:
done